src/services/api.service.ts

```diff
@@ -57,7 +57,7 @@ export const getPlaylistById = async (id?: string, params: GetPlaylistParams = {
   }
 
   const pathname = drmPolicyId ? `/v2/playlists/${id}/drm/${drmPolicyId}` : `/v2/playlists/${id}`;
-  const url = addQueryParams(`${import.meta.env.APP_API_BASE_URL}${pathname}`, params);
+  const url = addQueryParams(useConfigStore.getState().apiHost + pathname, params);
```

Comment:
```suggestion
  const url = addQueryParams(useConfigStore(s => s.apiHost) + pathname, params);
```

Reply:
We can't use hooks this way here. Zustand provides non-reactive approach (applied here). Functions receive updated values when being invoked.

Reply:
Oh, you're right. Ignore all these suggestions then.